lib/MusicBrainz/Server/Entity/Barcode.pm

```diff
@@ -14,6 +14,7 @@ sub type {
     return 'EAN' if length($self->code) == 8;
     return 'UPC' if length($self->code) == 12;
     return 'EAN' if length($self->code) == 13;
+    return 'GTIN' if length($self->code) == 14;
```

Comment:
I guess the idea is that a GTIN can be less than 14 digits, but if it's 14 digits then it's definitely a GTIN?

Reply:
Pretty much.

Reply:
Fun fact: GTIN was previously named [EAN-14](http://www.transbar.fr/choisir/choisir_ean14.html?lg=en).

This subroutine is dead code anyway. To be removed in PR https://github.com/metabrainz/musicbrainz-server/pull/3155 / commit https://github.com/metabrainz/musicbrainz-server/pull/3155/commits/25e436078251648727d4b111596684d5a853cf34.